pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1576,4 +1576,12 @@ self: super: {
     extraPrefix = "";
   }));
 
+  # 2020-06-24: Jailbreaking because of restrictive test dep bounds
+  # https://github.com/NixOS/nixpkgs/pull/91319#issuecomment-647844448
```

Comment:
Did you mean for this to be a link to the upstream trial library?

Reply:
Oh, wow. It was late …

Reply:
Fixed